src/components/v5/common/Fields/InputBase/InputBase.tsx

```diff
@@ -32,7 +33,7 @@ const InputBase: FC<InputBaseProps> = ({
 
   return (
     <div className={wrapperClassName}>
-      {prefix && prefix}
+      {prefix && <span className={prefixClassName}>{prefix}</span>}
```

Comment:
Just add this span to `prefix` outside of this component

Reply:
the prefix is passed like that, to I guess the span should be inside this component or I'm wrong
<img width="481" alt="Screen Shot 2023-10-12 at 09 51 22" src="https://github.com/JoinColony/colonyCDapp/assets/119587095/747a0366-187d-41eb-bbfd-8b98db532755">

Reply:
Prefix and suffix are reactNodes so you can pass whatever you want